defaults/main.yml

```diff
@@ -325,6 +325,13 @@ docker__pki_key: 'default.key'
 # Firewall and ferment support [[[
 # --------------------------------
 
+# .. envvar:: docker__ferm_post_hook [[[
+#
+# Enable or disable installation for the ``ferm`` post hook when :program:`ferment`
+# is disabled.
+docker__ferm_post_hook: '{{ ((ferm__enabled|d() | bool) and (docker__ferment|d() | bool == False)) }}'
```

Comment:
You should use Ansible local facts to check `debops.ferm` status (`ferm__enabled` might not always be available, which might change the status of this variable). Actually you don't even need to rely on the actual enabled/disabled status of `ferm`, only if the `debops.ferm` role was configured. The `docker__ferment` variable is a `debops.docker` default variable, therefore it's always present and you don't need to check if it's defined.

I would rewrite this condition like this:
```yaml
docker__ferm_post_hook: '{{ True
                            if (ansible_local|d() and ansible_local.ferm|d() and
                                docker__ferment|bool) else False }}'
```
Can you check if this works and update the PR if it does?